Makefile

```diff
@@ -6,3 +6,17 @@ pytest:
 
 jstests:
 	npx karma start karma.conf.js
+
+docker:
+	docker-compose up
```

Comment:
I propose `docker-compose up --force-recreate` it helps to avoid bugs that emerge when re-using a container whose state has drifted